### DISCUSSION TITLE
Question to 'User friendly failure messages' (jira#CFSA-1774)

### DISCUSSION THREAD

**Discussion by angelabriel** [trento-project/wanda#190] [Q&A]
Will it be possible to use 'rhai' expressions in the 'failure_msg:'?
e.g. something like
`failure_msg: SBD devices count was expected to be ${values.expected_multiple_sbd_device} but configured value is ${facts.sbd_multiple_sbd_device.split(";").len()}`

**Comment by fabriziosestito** [comment]
Yes, this was proposed. We will consider it while refining the card in question.

**Comment by angelabriel** [comment]
Thanks
